core/templates/pages/admin-page/misc-tab/admin-misc-tab.directive.html

```diff
@@ -61,6 +61,21 @@ <h3>Regenerate contribution opportunities for a topic:</h3>
   </div>
 </md-card>
 
+<md-card class="oppia-admin-page-card oppia-page-card oppia-long-text">
+  <h3>Get Exploration Interactions</h3>
+  <div class="oppia-query-form">
+    <form class="form" ng-submit="$ctrl.getExplorationInteractions()">
+      <div class="form-group row">
+        <label>Exploration ID:</label> <input type="text" ng-model="$ctrl.explorationId" required="true">
+      </div>
+      <div class="form-group row">
+        <input type="submit" value="submit query">
+      </div>
+    </form>
+    <p ng-if="$ctrl.explorationInteractionIds"><[$ctrl.explorationInteractionIds]></p>
```

Comment:
Might we be able to list this as a bullet list instead of a comma separated list? I think that might be a bit more consumable.

Reply:
Also how should i use ng repeat here I'm a bit confused 😅

Reply:
I believe you can use ``ng-repeat`` for this. Here's one example: https://github.com/oppia/oppia/blob/ae649aa08f1375457ec9e3c90257197b68fec7cd/core/templates/pages/admin-page/jobs-tab/admin-jobs-tab.directive.html#L13.

Also the Angular documentation has more samples: https://docs.angularjs.org/api/ng/directive/ngRepeat.

Reply:
Done thanks